### PR TITLE
[docs] OIDC: Add support for arbitrary claims for authorization subjects

### PR DESCRIPTION
**Pull request by vhdirk**
Documentation for #974

### CODE REVIEW

File: documentation/src/main/resources/pages/ditto/installation-operating.md [hunk 1]
```diff
@@ -88,43 +88,64 @@ proxy_set_header              x-ditto-pre-authenticated "nginx:${remote_user}";
...
     oauth {
       openid-connect-issuers = {
-        myprovider = "localhost:9000"
+        myprovider = {
+          issuer = "localhost:9000"
+          auth-subjects = [
+            "{{ jwt:sub }}",
+            "{{ jwt:sub }}/{{ jwt:scope }}",
+            "{{ jwt:sub }}/{{ jwt:scope }}@{{ jwt:client_id }}",
+            "{{ jwt:sub }}/{{ jwt:scope }}@{{ jwt:non_existing }}",
```
Comment (thjaeckle) [inline documentation/src/main/resources/pages/ditto/installation-operating.md:108]: I just learned that the "scope" field in JWT has the key "scp" and not "scope" - so we should adjust the example :)

```suggestion
            "{{ jwt:sub }}/{{ jwt:scp }}",
            "{{ jwt:sub }}/{{ jwt:scp }}@{{ jwt:client_id }}",
            "{{ jwt:sub }}/{{ jwt:scp }}@{{ jwt:non_existing }}",
```

File: documentation/src/main/resources/pages/ditto/installation-operating.md [hunk 2]
```diff
@@ -88,43 +88,64 @@ proxy_set_header              x-ditto-pre-authenticated "nginx:${remote_user}";
...
 }
 ```
 
 In order to do this by specifying a Java system property, use the following:
 
-```
--Dditto.gateway.authentication.oauth.openid-connect-issuers.myprovider=localhost:9000
+```shell
+-Dditto.gateway.authentication.oauth.openid-connect-issuers.myprovider.issuer=localhost:9000
+-Dditto.gateway.authentication.oauth.openid-connect-issuers.myprovider.auth-subjects.0='{{ jwt:sub }}/{{ jwt:scope }}'
```
Comment (thjaeckle) [inline documentation/src/main/resources/pages/ditto/installation-operating.md:121]:
```suggestion
-Dditto.gateway.authentication.oauth.openid-connect-issuers.myprovider.auth-subjects.0='{{ jwt:sub }}/{{ jwt:scp }}'
```

**Review by thjaeckle — CHANGES_REQUESTED** [review]
Added inline comments.

Could you also add a section to the "Migration notes" for the upcoming 2.0.0 release? We already add them here:
https://github.com/eclipse/ditto/blob/master/documentation/src/main/resources/pages/ditto/release_notes_next.md

File: documentation/src/main/resources/pages/ditto/installation-operating.md (hunk 1 above)
Comment (vhdirk) [reply documentation/src/main/resources/pages/ditto/installation-operating.md:108]: Ah, didn't know that. thanks!

**Review by thjaeckle — APPROVED** [review]
LGTM, thanks for the docs as well 👍

**Pull request merged**